### ISSUE TITLE
The adversarial loss

### ISSUE THREAD

**Issue by czjghost** [Lornatang/ESRGAN-PyTorch#6]
![YL9B6ROC(0MA@G1$H(P_08S](https://user-images.githubusercontent.com/84562125/125883222-50327081-077c-4790-86bd-f2147530db68.png)
Thanks very much for your work, I haved benefit much from it ~ But I found your adversarial loss is a little different from the paper.  Maybe it is similiar to SRGAN's adversarial loss ? just like make fake_img more realistic? I found ESRGAN also make real_img less realistic... Or I misunderstand it ?

I want to change it to make it the same as the paper, like this change in your code:

adversarial_loss = adversarial_criterion(fake_output - torch.mean(real_output), real_label) + 
                             adversarial_criterion(real_output - torch.mean(fake_output), fake_label)

Is it right? Thank you in advance for your help

**Comment by czjghost** [comment]
adversarial_loss = adversarial_criterion(fake_output - torch.mean(real_output), real_label) + adversarial_criterion(real_output - torch.mean(fake_output), fake_label)

**Comment by Lornatang** [comment]
It look right

**Comment by czjghost** [comment]
thanks
